pkg/mount/dag_steps.go

```diff
@@ -354,14 +348,19 @@ func (s *State) WriteSentinelDagStep(g *herd.Graph, deps ...string) error {
 		herd.WithCallback(func(ctx context.Context) error {
 			var sentinel string
 
+			internalUtils.Log.Info().Msg("Will now create /run/cos is not exists")
 			err := internalUtils.CreateIfNotExists("/run/cos/")
 			if err != nil {
+				internalUtils.Log.Err(err).Msg("failed to create /run/cos")
 				return err
 			}
-			runtime, err := state.NewRuntime()
+
+			internalUtils.Log.Info().Msg("Will now create the runtime object")
```

Comment:
I rather set several of this to debug.
The idea is to have immucore on debug while we work on this but then move it to info once its deemed more production ready for uki, so having all those messages seems a bit overboard

Reply:
Done.